plugins/test/unit/plugins/catalogers/test_yum.py

```diff
@@ -53,7 +53,7 @@ def test_metadata(self):
         expected = {
             'id': TYPE_ID,
             'display_name': "Yum Cataloger",
-            'types': [models.RPM.TYPE]
+            'types': [ids.TYPE_ID_RPM]
```

Comment:
Better to use `models.RPM._content_type_id` instead of constant in `ids` module?